frontend/rust-lib/dart-ffi/src/lib.rs

```diff
@@ -110,8 +110,14 @@ async fn post_to_flutter(response: AFPluginEventResponse, port: i64) {
 }
 
 #[no_mangle]
-pub extern "C" fn get_logtrace_from_frontend(logtrace: *const c_char) {
+pub extern "C" fn log(level: i64, logtrace: *const c_char) {
     let c_str = unsafe { CStr::from_ptr(logtrace) };
     let str_slice = c_str.to_str().unwrap();
+    match level {
+        0 => tracing::trace!("{}", logtrace),
```

Comment:
tracing has these levels: info, debug, trace, warn, and error. I think we can add these too. You can create an Enum for this. Then you can use a match pattern.

Reply:
i think i got what you mean, but i got compilation error when try to compare "level" vs enum in 'match'
 ```
   match level {
        0 => tracing::info!("{}", str_slice),
        // LOGLEVEL::INFO as i64 => tracing::info!("{}", str_slice),
        // LOGLEVEL::INFO  => tracing::info!("{}", str_slice),
```